src/app/cli/src/client.ml

```diff
@@ -466,6 +466,23 @@ let send_payment =
   user_command body ~label:"payment" ~summary:"Send payment to an address"
     ~error:"Failed to send payment"
 
+let change_delegate =
```

Comment:
This function exists as `delegate_stake` already (right below). The docs you wrote are more descriptive though. Can you just delete `delegate_stake`?

Reply:
whoops, thanks for the heads up